lighthouse-cli/bin.ts

```diff
@@ -293,9 +293,11 @@ function saveResults(results: Results,
           return innerPromise.then((_: Results) => Printer.write(results, outputType, outputPath));
         }, Promise.resolve(results));
       } else {
-        const outputPath = flags.outputPath || `${resolvedPath}.report.${flags.output}`;
+        let outputPath = flags.outputPath || `${resolvedPath}.report.${flags.output}`;
+        outputPath = outputPath.replace(/\.domhtml$/, '.html');
```

Comment:
this is kind of a stretch, but since we support multiple output modes and you added support for letting lighthouse output both the html and the dom reports below, would it make sense to do `outputPath = outputPath.replace(/\.domhtml$/, '.dom.html');` or something so that they aren't written to the same file name?

To be fair, the only use case I can see for that is for development of the new report, but it might be helpful to be able to output both while we work on it